src/Spice86.Core/Emulator/InterruptHandlers/Input/Mouse/MouseDriver.cs

```diff
@@ -251,4 +258,27 @@ public class MouseDriver : IMouseDriver {
     }
 
     private record MouseRegisters(ushort Es, ushort Ds, ushort Di, ushort Si, ushort Bp, ushort Sp, ushort Bx, ushort Dx, ushort Cx, ushort Ax);
+    
+    /// <inheritdoc />
+    public SegmentedAddress WriteAssemblyInRam(MemoryAsmWriter memoryAsmWriter) {
+        // Mouse driver implementation:
+        //  - Create a callback (0xFE) that will call the BeforeUserHandlerExecution method
+        //  - Create a Far call instruction that is calling a dummy handler by default, but can be modified to call anything
+        //  - Create a callback (0xFF) that does the cleanup
+        //  - Create a FAR ret
+        //  - Make the far call point to the FAR ret so that without any setup we call a user routine that does nothing
+        
+        // Write ASM
+        SegmentedAddress driverAddress = new SegmentedAddress(memoryAsmWriter.CurrentAddress);
+        memoryAsmWriter.RegisterAndWriteCallback(BeforeUserHandlerExecutionCallbackNumber, BeforeUserHandlerExecution);
+        // Far call to nothing for now, we will overwrite the address to the default handler later
```

Comment:
'Does nothing for now' ?

Reply:
Reworked the code to make the intent less cryptic...